src/NServiceBus.Persistence.AzureTable/SagaPersisters/AzureSagaPersister.cs

```diff
@@ -12,12 +12,12 @@
 
     class AzureSagaPersister : ISagaPersister
     {
-        public AzureSagaPersister(IProvideCloudTableClient tableClientProvider, bool autoUpdateSchema, bool migrationModeEnabled, bool assumeSecondaryIndicesExist = false, bool assumeSecondaryKeyUsesANonEmptyRowKeySetToThePartitionKey = false)
+        public AzureSagaPersister(IProvideCloudTableClient tableClientProvider, bool autoUpdateSchema, bool migrationModeEnabled, SecondaryIndex secondaryIndices)
```

Comment:
We should probably fix the name from `secondaryIndices` (plural) to `secondaryIndex` (singular).